client/jetpack-connect/plans-landing.jsx

```diff
@@ -106,7 +106,7 @@ class PlansLanding extends Component {
 				>
 					<PlansSkipButton onClick={ this.handleSkipButtonClick } />
 					<LoggedOutFormLinks>
-						<JetpackConnectHappychatButton>
+						<JetpackConnectHappychatButton eventName="calypso_jpc_plansanding_chat_initiated">
```

Comment:
You probably meant `calypso_jpc_planslanding_chat_initiated` here.

Reply:
right! good catch :+1: